src/megameklab/com/ui/view/CVChassisView.java

```diff
@@ -330,7 +330,7 @@ private void refreshTurrets() {
             }
         } else {
             cbTurrets.addItem(TURRET_SINGLE);
-            if (techManager.isLegal(TA_DUAL_TURRET)) {
+            if (!EntityMovementMode.WIGE.equals(getMovementMode()) && techManager.isLegal(TA_DUAL_TURRET)) {
```

Comment:
I'll assume that the techManager.isLegal() method doesn't have access to the movement mode.

Reply:
The tech manager only deals with intro years, tech base, and tech level.